test_extras/test_chemprop/test_chemprop_pipeline.py

```diff
@@ -306,3 +335,44 @@ def test_prediction(self) -> None:
 
         self.assertEqual(proba.shape, proba_copy.shape)
         self.assertTrue(np.allclose(proba[~nan_indices], proba_copy[~nan_indices]))
+
+
+class TestMulticlassClassificationPipeline(unittest.TestCase):
+    """Test the Chemprop model pipeline for classification."""
+
+    def test_prediction(self) -> None:
+        """Test the prediction of the classification model."""
+
+        test_data_df = pd.read_csv(
+            TEST_DATA_DIR / "multiclass_mock.tsv", sep="\t", index_col=False
+        )
+        print(test_data_df.head())
+        print(test_data_df.columns)
+        classification_model = get_multiclass_classification_pipeline()
+        mols = test_data_df["Molecule"].tolist()
+        classification_model.fit(
+            mols,
+            test_data_df["Label"].to_numpy(),
+        )
+        pred = classification_model.predict(mols)
+        proba = classification_model.predict_proba(mols)
+        self.assertEqual(len(pred), len(test_data_df))
+        self.assertEqual(proba.shape[1], 3)
+        self.assertEqual(proba.shape[0], len(test_data_df))
+
+        model_copy = joblib_dump_load(classification_model)
+        pred_copy = model_copy.predict(mols)
+        proba_copy = model_copy.predict_proba(mols)
+
+        nan_indices = np.isnan(pred)
```

Comment:
`np.isnan` rather returns a mask and not indices. The name is not correct.

Reply:
again @c-w-feldmann should we change this for Classification Pipeline as well?

Reply:
yep